changelog.txt

```diff
@@ -1,5 +1,12 @@
 *** Sensei Changelog ***
 
+2016.05.05 - version 1.9.6
```

Comment:
We'll add this later, feel free to add this in as a comment to the PR, but adding this is causing merge conflicts.